testcases/kernel/syscalls/lgetxattr/lgetxattr02.c

```diff
@@ -19,6 +19,21 @@
 * 3) lgetxattr(2) should return -1 and set errno to EFAULT.
 */
 
+/*
+ * Patch Description:
+ * Test failure reason in SGX-LKL:
+ * [[  SGX-LKL ]] libc_start_main_stage2(): Calling app main: /ltp/testcases/kernel/syscalls/lgetxattr/lgetxattr02
+ * tst_test.c:1106: INFO: Timeout per run is 0h 05m 00s
+ * tst_test.c:1125: INFO: No fork support
+ * lgetxattr02.c:78: CONF: no xattr support in fs or mounted without user_xattr option
+ *
+ * Workaround to fix the issue:
+ * Modified the tests to use root filesystem.
+ * Commented a test, which needs to be enabled once git issue 297 is fixed
+ * Issue 297: [Tests] lkl_access_ok() should return -1 on invalid access
+ * https://github.com/lsds/sgx-lkl/issues/297
+ */
+
```

Comment:
Change to:

* Patch Description:
  Test failure reason in SGX-LKL:
  [[  SGX-LKL ]] libc_start_main_stage2(): Calling app main: /ltp/testcases/kernel/syscalls/lgetxattr/lgetxattr02
  tst_test.c:1106: INFO: Timeout per run is 0h 05m 00s
  tst_test.c:1125: INFO: No fork support
  lgetxattr02.c:78: CONF: no xattr support in fs or mounted without user_xattr option
 
 * Workaround to fix the issue:
  Modified the tests to use root filesystem.
  Disabled a test, which needs to be enabled once git issue https://github.com/lsds/sgx-lkl/issues/297 is fixed